doc/introduction.rst

```diff
@@ -0,0 +1,103 @@
+Introduction
+============
+
+Simpleline is a text UI framework. Originally a part of the Anaconda installer project.
+
+This UI is simple and easy to use. It is designed to be used with line-based machines and tools
+(e.g. serial console) so that every new line it appended to the bottom of the screen. Printed lines
+are never rewritten!
+
+How to use
+----------
+
+The best learning sources can be found in the
+`examples directory <https://github.com/rhinstaller/python-simpleline/tree/master/examples>`_ in the
+`GitHub repository <https://github.com/rhinstaller/python-simpleline>`_ and you can read the
+:ref:`guide_label` section of this documentation. However, the basic usage of the Simpleline will be
+shown here too to get the idea how Simpleline works::
```

Comment:
`the idea` -> `and idea of`

Reply:
I guess it should be `an` instead of `and`